registry.json

```diff
@@ -3350,5 +3350,20 @@
     "telegram": "heliastaking",
     "operatorName": "Miro",
     "github": "heliastaking"
+  },
+  {
+    "ticker": "MSTR",
+    "poolId": "abea730f5c32157c76c876597d0b7093d50754d0274d67c62d7434b9",
+    "social": {
+      "facebook": "",
+      "twitter": "MStakepool",
+      "website": "http://www.maestro-pool.com",
+      "reddit": "https://www.reddit.com/user/MAESTRO-stake-pool",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "MAESTRO_Pool",
+    "operatorName": "MGAV",
```

Comment:
Please make this your real full name, real first name or leave blank

Reply:
@Bez625 done!